BUILD.bazel

```diff
@@ -46,6 +49,69 @@ container_bundle(
     },
 )
 
+container_bundle(
+    name = "e2e-env-docker-loadable",
+    images = {
+        "{STABLE_IMG_REGISTRY}/{STABLE_IMG_REPOSITORY}/cip-e2e-env:latest": "//:e2e-env",
+        "{STABLE_IMG_REGISTRY}/{STABLE_IMG_REPOSITORY}/cip-e2e-env:{IMG_TAG}": "//:e2e-env",
+    },
+)
+
+toolchain_container(
+    name = "e2e-env",
+    base = "@bazel-docker-gcloud//image",
+    cmd = [
+        "/bin/sh",
+        "-c",
+        "/bin/bash",
```

Comment:
What is this doing?

Reply:
It uses /bin/bash as the entrypoint instead of /bin/sh.

But I realize now that the base image we use (`gcr.io/asci-toolchain/nosla-ubuntu16_04-bazel-docker-gcloud`) already uses bash as the default command, so this bit is redundant. Will remove.